labelCloud/io/segmentations/base.py

```diff
@@ -0,0 +1,58 @@
+from abc import ABC, abstractmethod
```

Comment:
Please sort your imports.

I use vscode's "sort imports" but if you use another IDE you could also use isort for example (https://pycqa.github.io/isort/).

Reply:
sorted here: [058b502](https://github.com/ch-sa/labelCloud/pull/92/commits/058b5023af21d267eb7232a8ab5176a469aa2812)